### PR TITLE
[vcpkg] Call vcpkg_copy_pdbs in each vcpkg install functions.

### PR DESCRIPTION
**Pull request by JackBoosY**
I remember we have a plan for this.

### CODE REVIEW

**Comment by Neumann-A** [comment]
Only if the warning is somehow silenced. If there is no PDB in the build dir there is no reason to assume the libs have ever been build with /Zi or similar.

**Comment by JackBoosY** [comment]
@Neumann-A  I assume `vcpkg_copy_pdbs` will check whether the pdb file is exist first?

**Comment by JackBoosY** [comment]
> > @Neumann-A I assume vcpkg_copy_pdbs will check whether the pdb file is exist first?
> 
> The problem is it creates a message for every not found pdb for any lib. I doubt it does any checking yet.

Well, how about remove this message?

**Comment by Neumann-A** [comment]
> Well, how about remove this message?

I don't mind removing it. Don't know how others feel about that so. There is some reasoning behind having it. I just don't want to have the logs unjustified spammed with warnings about missing pdbs which never were build.

**Comment by JackBoosY** [comment]
> > Well, how about remove this message?
> 
> I don't mind removing it. Don't know how others feel about that so. There is some reasoning behind having it. I just don't want to have the logs unjustified spammed with warnings about missing pdbs which never were build.

More appropriate, we should add this feature in the post check of vcpkg-tool.

File: scripts/cmake/vcpkg_copy_pdbs.cmake [hunk 1]
```diff
@@ -12,8 +12,6 @@ function(vcpkg_copy_pdbs)
         )
     endif()
 
-    set(dlls_without_matching_pdbs "")
```
Comment (JackBoosY) [inline scripts/cmake/vcpkg_copy_pdbs.cmake:15]: This changes should wait for vcpkg-tool to add new post check.

Comment (JackBoosY) [reply scripts/cmake/vcpkg_copy_pdbs.cmake]: I think we can keep this changes here since it will only shows when generated dlls.

**Comment by JackBoosY** [comment]
@BillyONeal Seems the changes does not fix the CI issue.

**Comment by JackBoosY** [comment]
I will separate this PR to 10 PRs to archive this.

**Comment by JackBoosY** [comment]
Close this PR since it has conflict with the current solution.

**Pull request closed without merging**